### PR TITLE
revive CODEOWNERS

### PR DESCRIPTION
**Pull request by valpinkman**
### 📝 Description

This PR uses the previous CODEOWNERS to allow each team to be pinged on specific part of the code.
If some of the rules are not necessary for your team, please provide a fix on this PR directly.

⚠️ Right  now this CODEOWNERS just serves as a notification status. No PR will be  blocked by the codeowner status

### ❓ Context

- **Impacted projects**: `monorepo` 
- **Linked resource(s)**:  [LedgerHQ/ledger-live-issues#3](https://github.com/LedgerHQ/ledger-live-issues/issues/3) 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

No Demo

### 🚀 Expectations to reach

Allows team to be pinged when specific part of code is changed.
Closes https://github.com/LedgerHQ/ledger-live-issues/issues/3

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by haammar-ledger — CHANGES_REQUESTED** [review]

**Review by gre — APPROVED** [review]

File: libs/ledger-live-common/CODEOWNERS [hunk 1]
```diff
@@ -1,9 +0,0 @@
-*                 @ledgerhq/live-blockchain-support
-.github           @ledgerhq/live-devx
-src/hw            @ledgerhq/live-devices
-src/manager       @ledgerhq/live-devices
-src/apps          @ledgerhq/live-devices
-src/notifications @ledgerhq/live-hub
-src/featureFlags  @ledgerhq/live-hub
```
[the remark this answers is not in the document]
Comment (gre) [reply libs/ledger-live-common/CODEOWNERS:7]: to me, blockchain team should only own "live-common src families" folder, the other folders are gray area shared by all teams

[the remark this answers is not in the document]
Comment (valpinkman) [reply libs/ledger-live-common/CODEOWNERS:7]: @gre @haammar-ledger feel free to update the PR

**Review by haammar-ledger — APPROVED** [review]
Proposed changes : https://github.com/LedgerHQ/ledger-live/pull/1049/commits/78c4e727137c8947a643549e3e1693a496bc1f1e
- More specific scope for blockchain team
- Include QA team on the bot config files
- Exceptionally assign the whole `ledger-live` team for changes on `@ledgerhq/types-live` because they're relevant to everyone

**Comment by haammar-ledger** [comment]
Also assigning `ledger-live` as a reviewer since (unlike most PRs) this change is relevant to the full team. :slightly_smiling_face:

**Comment by elbywan** [comment]
> Exceptionally assign the whole ledger-live team for changes on @ledgerhq/types-live because they're relevant to everyone

@haammar-ledger No please, they are not relevant to the devx team and I would like not to get pinged.

**Review by elbywan — CHANGES_REQUESTED** [review]
See above comment.

**Comment by haammar-ledger** [comment]
> > Exceptionally assign the whole ledger-live team for changes on @ledgerhq/types-live because they're relevant to everyone
> 
> @haammar-ledger No please, they are not relevant to the devx team and I would like **not** to get pinged.

@jelbaz-ledger Got it, sorry for not thinking about your use-case. Fixed :+1: please re-review.

**Review by elbywan — APPROVED** [review]
> @elbywan Got it, sorry for not thinking about your use-case. Fixed 👍 please re-review.

No worries, thanks for fixing 👍

**Review by gre — APPROVED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ledgerjs/packages/cryptoassets                                  @ledgerhq/live-blockchain-support
+ledgerjs/packages/types-cryptoassets                            @ledgerhq/live-blockchain-support
+ledgerjs/packages/hw-app-*                                      @ledgerhq/live-blockchain-support
+
+# Live Platform and Services Team
+
+ledger-live-desktop/src/renderer/screens/exchange/  @ledgerhq/live-platform-and-services
+ledger-live-mobile/src/screens/Exchange/            @ledgerhq/live-platform-and-services
+ledger-live-common/src/exchange/                    @ledgerhq/live-platform-and-services
+ledger-live-common/src/platform/                    @ledgerhq/live-platform-and-services
```
Comment (Justkant) [inline CODEOWNERS:23]: Should we separate the platform team and the services team in github as we're now different teams ?
Or is it only for the platform team and we just need to rename the team ?

Comment (valpinkman) [reply CODEOWNERS:23]: Valid point. Whenever those teams are created we can update

**Pull request merged**